main/main.cpp

```diff
@@ -44,9 +50,44 @@ static void initResources()
 
 int main(int argc, char** argv)
       {
+      // Force the 8-bit text encoding to UTF-8. This is the default encoding on all supported platforms except for MSVC under Windows, which
+      // would otherwise default to the local ANSI code page and cause corruption of any non-ANSI Unicode characters in command-line arguments.
+      QTextCodec::setCodecForLocale(QTextCodec::codecForName("UTF-8"));
+
       initResources();
 
       ModulesSetup::instance()->setup();
 
-      return Ms::runApplication(argc, argv);
+#if (defined (_MSCVER) || defined (_MSC_VER))
+      // On MSVC under Windows, we need to manually retrieve the command-line arguments and convert them from UTF-16 to UTF-8.
+      // This prevents data loss if there are any characters that wouldn't fit in the local ANSI code page.
+      int argcUTF16 { 0 };
```

Comment:
Please use `=` for initializations.

Reply:
Oops, didn't see there're already discussions

Reply:
But I guess `=` initialization is still way more common than braces in MuseScore code, so I think it's best to use that.

Reply:
And compaitible with C and C++ programmers ;-)